docs/fr/delivery-management.md

```diff
@@ -30,7 +30,7 @@ Mon panier contient 4 lignes articles.
 Dans le cadre d'une configuration "Flat fees" de 10€, le prix des frais de port sera de 10€.
 Dans le cadre d'une configuration "Flat fees" de 10€ avec l'option "Apply flat fee for each item instead of once for the whole order", le prix des frais de port sera de 4 lignes articles * 10€, soit 40€.
 
-### Article Standalone
+### Article Indépendant
```

Comment:
"Standalone" (even in french) is a specific e-commerce wording, we keep it as it is.
You can remove this correction.